package.json

```
@@ -18,7 +18,8 @@
     "integration-coverage": "nyc mocha -r source-map-support/register -r ts-node/register --full-trace -t 600000 -b false \"integration_test/**/*.spec.ts\" && nyc report | coveralls",
     "doc": "rm -rf ./tsdoc && typedoc --readme none --excludeNotExported --mode file --theme markdown --out ./tsdoc ./src",
     "compile-tezos-parsers": "nearleyc grammar/tezos/Micheline.ne -o src/chain/tezos/lexer/Micheline.ts && nearleyc grammar/tezos/Michelson.ne -o src/chain/tezos/lexer/Michelson.ts && nearleyc grammar/tezos/MichelsonParameters.ne -o src/chain/tezos/lexer/MichelsonParameters.ts && nearleyc grammar/tezos/EntryPointTemplate.ne -o src/chain/tezos/lexer/EntryPointTemplate.ts",
-    "release-check": "npm run test && npm audit && npm outdated"
+    "release-check": "npm run test && npm audit && npm outdated",
+    "postinstall": "npm run package"
```

le `postinstall` n'est utile qu'en environnement de dev. On lance `npm run package` à la main avant de publier le paquet 

```suggestion
    "release-check": "npm run test && npm audit && npm outdated"
```